Code/GraphMol/RGroupDecomposition/RGroupMatch.h

```diff
@@ -33,8 +33,8 @@ struct RGroupMatch {
     auto rGroupsString = std::accumulate(
         rgroups.cbegin(), rgroups.cend(), std::string(),
         [](std::string s, const std::pair<int, RData>& rgroup) {
-          return std::move(s) + '|' + std::to_string(rgroup.first) + ':' +
-                 rgroup.second->toString();
+          return std::move(s) + "\n\t(" + std::to_string(rgroup.first) + ':' +
```

Comment:
This made the rgroups easier to read for me.  I can revert if necessary.

Reply:
I don't think it matters- it's just to see what is going on.